cli/cli.go

```diff
@@ -105,6 +105,21 @@ func Run(c *config.Config, name string, args []string) {
 	}
 }
 
+// Run the task with xtrace
+func RunTrace(c *config.Config, name string, args []string) {
+	task, ok := c.Tasks[name]
+	if !ok {
+		Fatalf("undefined task %q", name)
+	}
+
+	task.LookupPath = filepath.Dir(c.File)
+
+	err := task.RunTrace(args)
```

Comment:
it looks like this is the only line that differs from `Run`, why not just thread the `--xtrace` arg through to `Run` like the others and avoid having to duplicate things?